Exercise_9.R

```diff
@@ -0,0 +1,210 @@
+#Question 1
+
+library(ggplot2)
+
+ggplot(data)+
+  geom_point(aes(x=mutation, y=ponzr1Counts))
+
+ggplot(data)+
+  geom_boxplot(aes(x=mutation, y=ponzr1Counts))
+
+data <- read.csv("/Users/elizabethfortin12/Documents/ND First Year/Biocomputing/R_Programming/Exercise_9/ponzr1.csv")
```

Comment:
This should go up above the plots. And try to keep it coming from the raw data in the repo that way anyone could rerun your code. With the file path the way it is now, it would be hard to rerun.